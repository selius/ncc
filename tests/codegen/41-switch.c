int main()
{
	int i;

	i = 1;

	switch (i) {
	}

	switch (i) {
	case 1:
		__print_int(i);
	}

	switch (i) {
	case 1:
		__print_int(1);
		break;
	case 2:
		__print_int(2);
		break;
	default:
		__print_int(999);
	}

	i = 45;

	switch (i) {
	case 1:
		__print_int(i);
	}

	switch (i) {
	case 1:
		__print_int(1);
		break;
	case 2:
		__print_int(2);
		break;
	default:
		__print_int(999);
	}

	switch (i) {
	default:
		__print_int(999);
	}

	return 0;
}
