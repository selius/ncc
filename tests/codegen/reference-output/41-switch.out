1
1
999
999
